.changeset/unlucky-hornets-roll.md

```diff
@@ -0,0 +1,14 @@
+---
+"@clerk/clerk-js": patch
```

Comment:
❓  Should we go for `minor` here? Isn't this a new feature?

Reply:
Correct @anagstef, I will update the changeset 🙏

Reply:
Done